packages/clay-tabs/src/Content.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Receives a number that indicates the `tabkey` to be rendered.
+	 */
+	activeTabKey: number;
```

Comment:
`TabKey` is a bit strange. What about `activeIndex` since you are comparing it vs the array index.

Reply:
Makes sense, I'm going with `activeIndex`.